From 6c4643d61beea234ab69753e4b0d2276566499a0 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Thu, 14 Nov 2019 11:00:32 +0700
Subject: [PATCH 311/328] arm64: dts: renesas: hihope-common: add "shared-pin"
 property so SSI1

This patch adds "shared-pin" property to SSI1 to share clock line
with SSI{0,2,3,9}.
If not add this property, there is no sound output to SSI1.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-common.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/hihope-common.dtsi b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
index 7eff273..cd903c2 100644
--- a/arch/arm64/boot/dts/renesas/hihope-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
@@ -541,6 +541,10 @@
 	};
 };
 
+&ssi1 {
+	shared-pin;
+};
+
 &vspb {
 	status = "okay";
 };
-- 
1.9.1

